File: frontend/src/app/_model/courierDto.ts
export class CourierDto {
  courierId: string;
  cartId: number;
  date: string;
  status: string;


  constructor(courierId: string, cartId: number, date: string, status: string) {
    this.courierId = courierId;
    this.cartId = cartId;
    this.date = date;
    this.status = status;
  }
}

File: backend/src/main/java/com/eshop/backend/courier/dto/courierDto.java
package com.eshop.backend.courier.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class courierDto {
    public long cartId;
    public long courierId;
    public String date;
    public String status;
}
